docs/includes/introduction.txt

```diff
@@ -224,3 +224,11 @@ with only one command (recommended for **development environment only**)::
 
 
 3. Now you can add and manage your periodic tasks from the Django Admin interface.
+
+
+
+Working with django-celery-results
+-----------------------------------
+
+Now you can store PeriodicTask.name to django-celery-results (TaskResult.periodic_task_name).
+(more details here: https://github.com/celery/django-celery-beat/pull/477, https://github.com/celery/django-celery-results/pull/261)
```

Comment:
i think we should add more text and example with the links provided

Reply:
added, but i'm not familiar with .rst files, may you help to check it?